src/components/AuthorizationForm/AuthorizationForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { InputField, Button, T } from '@admiral-ds/react-ui';
+import { useNavigate } from 'react-router-dom';
+import GlobalFont from '../../assets/fonts/GlobalFont';
+import { FormWrapper, InputsWrapper } from '../WrappersS/Wrappers';
+import { useLoginMutation } from '../../services/apiService';
+import { routesEnum } from '../../routes/routesEnum';
+import { Link } from 'react-router-dom';
+import { useForm, Controller } from 'react-hook-form';
+
+interface FormValues {
+    username: string;
+    password: string;
+}
+
+export const AuthorizationForm = () => {
+    const navigateHome = useNavigate();
```

Comment:
```suggestion
    const navigate = useNavigate();
```
Это дефолтная функция react-router и если её искать будут, то по названию. Не надо её переименовывать, конечную точку роутинга укажет аргумент, переданный при вызове.

Reply:
fixed